src/main/kotlin/no/nav/syfo/dialogmote/DialogmotedeltakerService.kt

```diff
@@ -162,18 +162,26 @@ class DialogmotedeltakerService(
         if (brevIsReferat) {
             throw IllegalArgumentException("Cannot store response for referat")
         }
-
         return database.connection.use { connection ->
             val updateCount = connection.updateMotedeltakerArbeidstakerVarselRespons(
                 motedeltakerArbeidstakerVarselUuid = brevUuid,
                 svarType = svarType,
-                svarTekst = svarTekst,
+                svarTekst = sanitizeTextInput(svarTekst, 300),
             )
             connection.commit()
             updateCount > 0
         }
     }
 
+    private fun sanitizeTextInput(
+        tekst: String?,
+        truncate: Int,
+    ): String? {
+        return tekst?.let {
+            Regex("[^A-Za-z0-9 æøåÆØÅ%!()?.,]").replace(it.take(truncate), "_")
```

Comment:
Er det noen spesiell grunn til at f.eks. semikolon og kolon ikke er tillatt?
Hadde det vært enklere å definere en regex med ulovlige tegn, i stedet for å definere de vi godtar? 🤔

Reply:
Nei, ikke noen spesiell grunn - kan godt legge til semikolon og kolon. 

Men jeg synes det er greit å eksplisitt definere hva vi aksepterer for å unngå overraskelser. Hensikten er å unngå at noen klarer å bruke slike tekstfelt til å sende inn et script som blir kjørt i en eller annen sammenheng (selv om det er andre mekanismer som må svikte i tillegg for at det skal gjøre skade).

Reply:
Ja, det er et godt poeng!
Hva med bindestrek eller skråstrek?

Reply:
Fikset 👍 
Vi får legge til flere karakterer etter hvert som behovene blir avdekket.